docs/changelog.txt

```diff
@@ -145,8 +145,7 @@ Other Changes
 - Fix: String encoding with cases was non-deterministic and resulted in differing md5sums for the NewGRF.
 - Fix: Rearrange order of string id assignment to keep compatibility.
 - Fix: When some IDs depleted, an internal NML error was raised instead of the intended error message.
-- Fix: most_common_refit was reading the wrong bits in var 42; read the write bits and rename it to most_common_cargo_type
-       as it's more accurate; also add most_common_cargo_subtype which also reads var 42 and was previously missing
+- Fix: most_common_refit was reading the wrong bits in var 42; read the write bits and rename it to most_common_cargo_type as it's more accurate; also add most_common_cargo_subtype which also reads var 42 and was previously missing
```

Comment:
I removed the hard line-wrapping because the rest of the file doesn't use it, and I couldn't be bothered to handle it in the script.